lib/trino-hdfs/src/main/java/io/trino/filesystem/hdfs/HdfsFileIterator.java

```diff
@@ -75,11 +76,11 @@ public FileEntry next()
             path += relativeUri.getPath();
         }
 
-        List<BlockLocation> locations = Stream.of(status.getBlockLocations())
-                .map(HdfsFileIterator::toTrinoBlockLocation)
+        List<Block> locations = Stream.of(status.getBlockLocations())
```

Comment:
rename get locations?

Reply:
That's in `org.apache.hadoop.fs.LocatedFileStatus`